ui-admin/src/study/kits/KitEnrolleeSelection.tsx

```diff
@@ -279,13 +299,22 @@ export default function KitEnrolleeSelection({ studyEnvContext }: { studyEnvCont
       </div>
       <div className="d-flex align-items-center">
         <Link to={'../scan'}>
-          <Button variant="light" className="border m-1"><FontAwesomeIcon icon={faQrcode}/> Scan kit</Button>
+          <Button variant="light" className="border m-1"><FontAwesomeIcon icon={faQrcode}/> Scan in-person kit</Button>
         </Link>
         <Button onClick={() => {
-          setShowRequestKitModal(true)
+          setShowAssignKitModal(true)
         }}
         variant="light" className="border m-1" disabled={!enableActionButtons}
-        tooltip={enableActionButtons ? 'Send sample collection kit' : 'Select at least one participant'}>
+        tooltip={enableActionButtons
+          ? 'Assign a sample kit to a participant for manual kit shipment'
+          : 'Select at least one participant'}>
+          <FontAwesomeIcon icon={faBoxOpen} className="fa-lg"/> Assign return-only kit
```

Comment:
I don't love the verbiage. I want something like:
`Assign kit for manual shipment`
but that feels too verbose?